inst/rmarkdown/templates/ajs/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,221 @@
+\begin{figure}[hbt]
+\begin{center}
+\includegraphics{ajs-download20180210.pdf}
+\caption{\label{Fig3.4}Download statistics of articles per month at the Austrian Journal of Statistics.} 
+\end{center}
+\end{figure}
+
+\begin{table}[hbt]
+\caption{\label{Tab3.4}80 measured values of access time.}
+\vspace*{-5mm}
+\small
+\hspace{1.5cm}
+\begin{center}
+\begin{tabular}{|llllllll|}
+\toprule
+70.0 & 68.6 & 67.9 & 66.3 & 71.0 & 64.2 & 69.6 & 71.0\\
+... & ... & ... & ... & ... & ... & ... & ...\\
+%69.7 & 69.0 & 73.4 & 69.0 & 70.1 & 69.8 & 69.0 & 73.0\\
+%70.0 & 69.1 & 69.5-& 67.9 & 72.8 & 72.1 & 69.5+& 70.1\\
+%70.2 & 69.1 & 68.9 & 68.3 & 74.9 & 68.4 & 69.1 & 66.6\\
+%71.1 & 69.2 & 71.2 & 68.9 & 70.9 & 70.6 & 69.9 & 69.9\\
+%69.4 & 69.5+& 68.5+& 70.9 & 71.6 & 68.9 & 72.0 & 70.3\\
+%68.6 & 68.5-& 67.8 & 72.2 & 68.7 & 70.6 & 66.9 & 69.3\\
+%71.4 & 68.7 & 74.2 & 68.8 & 71.4 & 71.8 & 67.5-& 70.4\\
+%71.4 & 67.4 & 69.5-& 72.4 & 70.4 & 69.3 & 68.2 & 67.0\\
+71.7 & 70.5+& 72.5-& 68.2 & 67.6 & 68.6 & 70.5-& 65.8\\
+\bottomrule
+\end{tabular}
+\end{center}
+\end{table}
+
+Mathematical formulas should be enumerated as far as they are
+referenced \textbf{only}. E.g.~Equation~(\ref{form3.1}) shows the density 
+\begin{equation}\label{form3.1}
+f(x) =
+\frac1{\sqrt{2\pi}\sigma}e^{-\frac{(x-\mu)^2}{2\sigma^2}}\,.
+\end{equation}
+
+Please consider 
+
+- capitalized title case for titles and for all references and 
+- sentence, non-capitalized case for sections and subsections.
+
+
+
+Cited literature should be summarized at the end of the article in
+form of a non-enumerated section. \cite{leisch02} is a reference
+to an article. If the citation is in parentheses, it
+should look like \citep{leisch02}.
+References should be in title case.
+
+Correspondence addresses of author(s) should be added at the end
+of the manuscript. 
+
+
+# Submission and refereeing
+
+
+## Submission
+
+Consideration of the previously stated rules are of great help for
+the editing process. Whenever technically possible, manuscripts
+should be submitted through the editorial system at \href{www.ajs.or.at}{www.ajs.or.at}. The authors must be registered at this website.
+
+## Refereing
+
+All contributions will be anonymously refereed. Cited literature which is
+hardly available should accompany the submitted manuscript. It
+should also be considered to place used and analyzed data at
+disposal for the referees (if there are no legal or technical
+arguments against).
+
+Editor and referees must trust that the contribution has not been
+submitted for publication at the same time at another place. It is
+fair that the submitting author notifies if an earlier version has
+already been submitted somewhere before.
+
+## Copyrights and fees
+
+The Austrian Journal of Statistics is free of
+charge. Generally, we use the Creative Commons Attribution License (CC-BY).
+
+For copyright issues, publication ethics and malpractice statements, please have a look at 
+\href{www.ajs.or.at}{www.ajs.or.at}
```

Comment:
You could use markdown here too 

```suggestion
For copyright issues, publication ethics and malpractice statements, please have a look at 
<www.ajs.or.at>
```
which would lead to `\url`

or 

```suggestion
For copyright issues, publication ethics and malpractice statements, please have a look at 
[www.ajs.or.at](www.ajs.or.at)
```

which would lead to `\href{}{}`

Reply:
Much better solution, ok.